app/src/main/java/com/framgia/fsalon/screen/billdialog/BillDialogFragment.java

```diff
@@ -0,0 +1,85 @@
+package com.framgia.fsalon.screen.billdialog;
+
+import android.databinding.DataBindingUtil;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.DialogFragment;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.Window;
+import android.widget.ArrayAdapter;
+
+import com.framgia.fsalon.R;
+import com.framgia.fsalon.data.model.BillItemRequest;
+import com.framgia.fsalon.data.model.Service;
+import com.framgia.fsalon.data.model.Stylist;
+import com.framgia.fsalon.databinding.FragmentBilldialogBinding;
+import com.framgia.fsalon.screen.bill.BillViewModel;
+
+/**
+ * BillDialog Screen.
+ */
+public class BillDialogFragment extends DialogFragment {
+    private BillDialogContract.ViewModel mViewModel;
+    private ArrayAdapter<Stylist> mStylistAdapter;
+    private ArrayAdapter<Service> mServiceAdapter;
+    private BillItemRequest mBillItemRequest;
+    private Fragment mFragment;
+    private BillViewModel mViewModelBill;
+
+    public BillDialogFragment(
```

Comment:
FATAL: [ValidFragment] Avoid non-default constructors in fragments: use a default constructor plus `Fragment#setArguments(Bundle)` instead
 From the Fragment documentation:
*Every* fragment must have an empty constructor, so it can be instantiated when restoring its activity's state. It is strongly recommended that subclasses do not have other constructors with parameters, since these constructors will not be called when the fragment is re-instantiated; instead, arguments can be supplied by the caller with `setArguments(Bundle)` and later retrieved by the Fragment with `getArguments()`.